x-pack/legacy/plugins/ml/server/new_platform/job_service_schema.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+const analyzerSchema = {
+  tokenizer: schema.string(),
+  filter: schema.arrayOf(
+    schema.object({
+      type: schema.string(),
+      stopwords: schema.arrayOf(schema.maybe(schema.string())),
+    })
+  ),
+};
+
+export const catFieldExamplesSchema = {
```

Comment:
Might be worth calling this `categorizationFieldExamplesSchema` for clarity.

Reply:
Updated in 9d1014383ec15daef0ec6402379d54d412360d15